alerting/alerts.go

```diff
@@ -21,11 +22,17 @@ var (
 	ErrGetAlertGroupsBadPayload = fmt.Errorf("unable to retrieve alerts groups")
 )
 
-func (am *GrafanaAlertmanager) GetAlerts(active, silenced, inhibited bool, filter []string, receivers string) (apimodels.GettableAlerts, error) {
+type GettableAlerts = amv2.GettableAlerts
```

Comment:
There will be a better place to organise this, but for now, I want to make the diffs minimal until we know for sure that this works for Grafana. Then we can refactor away at our liking.